recipes/lmdb/all/conanfile.py

```diff
@@ -59,11 +61,8 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["lmdb"]
-        self.cpp_info.names["cmake_find_package"] = "LMDB"
-        self.cpp_info.names["cmake_find_package_multi"] = "LMDB"
-        self.cpp_info.names["pkg_config"] = "lmdb"
```

Comment:
most distros ship this file:

- https://packages.ubuntu.com/hirsute/amd64/liblmdb-dev/filelist 
- https://debian.pkgs.org/11/debian-main-amd64/liblmdb-dev_0.9.24-1_amd64.deb.html
- https://alpine.pkgs.org/3.14/alpine-main-aarch64/lmdb-dev-0.9.29-r0.apk.html
- https://fedora.pkgs.org/rawhide/fedora-x86_64/lmdb-devel-0.9.29-1.fc35.x86_64.rpm.html

Reply:
Distro are not a reference for pkg_config or cmake files, but well I can keep pkg_config name if you want.

Reply:
restored in https://github.com/conan-io/conan-center-index/pull/6411/commits/78fd2b5ee214958394f0b011fede8b5316efdaac